spacy_llm/models/rest/azure/model.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         self._deployment_name = deployment_name
         super().__init__(
             name=name,
-            endpoint=endpoint,
+            endpoint=endpoint or endpoint,
```

Comment:
Looks like a merge gone wrong? Do we want to just remove the edit?

Reply:
Yeah, this is odd.